src/content/bqtools-intro/steps/Step3.md

```diff
@@ -0,0 +1,41 @@
+<script>
+import Execute from "$components/Execute.svelte";
+import Link from "$components/Link.svelte";
+</script>
+
+<Link href="https://github.com/arcinstitute/binseq">`BINSEQ`</Link> is a file format for storing nucleotide sequences as binary data.
+It is designed to be efficient and compact, making it suitable for large-scale genomic data analysis.
+
+To convert a FASTQ (or FASTA) file to BINSEQ we can make use of the `bqtools` command-line tool.
+Specifically we can use `bqtools encode` to convert the file.
+
+Lets encode a single FASTQ file:
+
+<Execute command="bqtools encode fastq/sample1_R1.fastq.gz"/>
```

Comment:
When I run this, it seems the vbq file is not in the current directory but in the fastq/ folder, is that expected? I'm seeing that on my Mac and in the sandbox.bio environment.

Reply:
ah this was a mistake on my part. 

When `-o <path>` isn't included it'll just replace the current extension with `.vbq` or `.bq` so in this case it was writing to `fastq/sample1_R1.vbq`. 

I updated the example to write to an explicit path instead.